articles/azure/azs-how-create-service-fabric-portal.md

```diff
@@ -0,0 +1,218 @@
+---
+title: How to create a Service Fabric cluster | UKCloud Ltd
+description: Provides help for creating a Service Fabric cluster on UKCloud for Microsoft Azure
+services: azure-stack
+author: Bailey Lawson
+toc_rootlink: Users
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a Service Fabric cluster
+toc_fullpath: Users/How To/azs-how-create-service-fabric-portal.md
+toc_mdlink: azs-how-create-service-fabric-portal.md
+---
+
+# How to create a Service Fabric cluster using the Azure Stack portal
+
+## Overview
+
+Azure Stack Service Fabric is a distributed systems platform that makes it easy to package, deploy, and manage scalable and reliable microservices and containers. Service Fabric also addresses the significant challenges in developing and managing cloud native applications. Developers and administrators can avoid complex infrastructure problems and focus on implementing mission-critical, demanding workloads that are scalable, reliable, and manageable. Service Fabric represents the next-generation platform for building and managing these enterprise-class, tier-1, cloud-scale applications running in containers.
+
+The following process shows you how to setup a Service Fabric cluster in Azure using the Azure portal.
+
+## Prerequisites
+
+To complete the steps in this guide, you must have appropriate access to a subscription in the UKCloud Azure Stack portal.
+
+## Creating a new Key Vault
+
+Before creating a Service Fabric cluster, it is necessary to create a Key Vault to store the certificates for the cluster. These certificates are used in Service Fabric to provide authentication and encryption to secure various aspects of a cluster and its applications. More information on these certificates can be found [here](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-security#x509-certificates-and-service-fabric).

```

Comment:
Last sentence:

You can find more information about these certificates [here](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-security#x509-certificates-and-service-fabric).